lib/package-manager/yarn.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
I don't think we need this file. We can have a single install + test file that we pass an argument to. Since `packageManager.install` is already abstracted I don't think we need this extra boiler plate.